### ISSUE TITLE
[Feature Request] Separate Catalogues settings from Source Migration

### ISSUE THREAD

**Issue by GreatestFool** [Jays2Kings/tachiyomiJ2K#180]
Sources that are not hidden in *Catalogues* specify which are shown when you try to migrate a story to another source even if you've selected everyone in [*Select target sources*](https://i.imgur.com/JgXS35d.jpg) and [select *Search manually* in the context menu](https://i.imgur.com/tKJHVMn.jpg).

This is pretty bad from a user-standpoint. If I'm looking for new stories to add to *My Library*, I obviously go through the *Catalogues* however having over ten sources is just plain annoying, tedious and confusing at times, which is why I tend to limit it to the sources I tend to have the least problems with. If there is an issue with a story, like a missing chapter or similar, I then migrate to another source, here it is nice to have access to all possible sources because some of them have more chapters than others, some have better quality, etc. I'm honestly a little surprised searching manually after having chosen every source in *Select target sources* is separate from the normal catalogue.

Here's a couple of images that may explain what I'm trying to say better. [*Catalogues* with all sources shown and almost all sources selected in *Select target sources*.](https://i.imgur.com/6H73ZXe.png) [*Catalogues* with some hidden and almost all sources selected in *Select target sources*.](https://i.imgur.com/GFRWT7j.png)

### Requirements
Separating Migration's manual search from *Catalogues's* search. Could be done by having *Select target sources* as a temporary override for which sources are shown, and then default back to whatever the user has set as shown/hidden sources in their *Catalogues*.

**Comment by joseph619** [comment]
So you basically want source in `Source migration` to be limited to enabled sources in Catalogues?

**Comment by GreatestFool** [comment]
No, the opposite. That's currently how it's working. That's why I included [this picture](https://i.imgur.com/GFRWT7j.png), in hopes that it explained what I meant better than I did. If you look at the middle screenshot, you see all the sources I have extensions for and have enabled in _Select target sources_ when I'm migrating a manga, but since the only sources I don't have hidden in _Catalogues_ on the left are MangaDex, Mangahere, Mangasee and Webtoons, the result of [source migration's manual search](https://i.imgur.com/tKJHVMn.jpg) is only for those four sources and not for those I've chosen in _Select target sources._
